File: Ejes de Vehículos/Ejes de Vehículos.py
#define Hd
def mcd(a:int, b:int) -> int:
    return b if a%b==0 else mcd(b,a%b)

a:int
b:int
c:int
c = int(input())
c-=1
a = int(input())
while c:
    c-=1
    b = int(input())
    a = a*b // mcd(a,b)

print(a)

File: Ejes de Vehículos/Ejes de Vehículos.cpp
#include <stdio.h>
#define Hd main(void)
long long mcd(long long a,long long b){
    return (a%b==0? b:mcd(b,a%b));
}

Hd
{
    long long a,b,cp;
    scanf("%lld%lld",&cp,&a);
    cp--;
    while(cp--){
        scanf("%lld",&b);
        a=a*b/mcd(a,b);
    }
    printf("%lld",a);
}
